### ISSUE TITLE
Remove forms and calculate Camtrap DP spatial, temporal, and taxonomic scope

### ISSUE THREAD

**Issue by peterdesmet** [gbif/ipt#2047]
_First suggested in https://github.com/tdwg/camtrap-dp/issues/305._

Users currently have 2 ways of adding the spatial, temporal and taxonomic scope in a Camtrap DP:

1. Add this information manually in the metadata editor.
2. Upload a `datapackage.json` with that information.

The manually entry is tedious and can lead to inconsistencies (e.g. scope doesn't fit data) or incompleteness (e.g. only a few taxa provided). **I would therefore remove the entry forms in the IPT and calculate the scopes from the data.** That way users can:

1. ~Add this information manually in the metadata editor.~ Have the scopes calculated by the IPT.
2. Upload a `datapackage.json` to populate scope information (and not have it calculated). This also allows to provide richer information (see bold below) than what can be calculated from the data.

Additionally:

- The resource preview could still (calculate and) display the scope.
- The metadata editor could provide the option to remove a scope (and have it calculated).
- Alternatively, metadata editor could keep the forms, but add an option to calculate it from the data. Note that this opens the option for inconsistency again.

Scopes can be calculated from the data as follows:

- spatial: based on min/max deployment `latitude`/`longitude` to create a bounding box. **One can also express more complex shapes in geojson.**
- temporal: based on min/max deployment `deploymentStart`/`deploymentEnd`
- taxonomic: listing all unique `taxonID`/`scientificName`. **This can be completed with more information, such as higher classification, vernacular names etc.**

@timrobertson100 @mike-podolskiy90 thoughts?

**Comment by albenson-usgs** [comment]
It would be nice if it was this way for all data loaded in the IPT and not just CamtrapDP data. I'm hopeful that was the intent but the title of this issue makes me think it wasn't.

**Comment by peterdesmet** [comment]
@albenson-usgs I totally agree, but purposefully kept the scope to Camtrap DP as that is easier to implement (is a new feature, doesn't disrupt current use) and because Camtrap DP dictates the scopes to be exhaustive (intent, but not the case for EML). Maybe we should create a separate issue for EML-based resources?

**Comment by albenson-usgs** [comment]
> It would be nice if it was this way for all data loaded in the IPT and not just CamtrapDP data. I'm hopeful that was the intent but the title of this issue makes me think it wasn't.

Please ignore this comment. I was out of date. I see in the newest update of the IPT that it is now possible to infer these from the source data.

**Comment by peterdesmet** [comment]
@albenson-usgs oh, you are right, I didn't know about that feature 👍 :

<img width="877" alt="Screenshot 2023-06-15 at 12 19 39" src="https://github.com/gbif/ipt/assets/600993/0df3b752-c284-4f43-b475-b38967e092df">

For Camtrap DP:

- It would be `Automatically infer from source data` or `As provided in uploaded datapackage.json`
- The preview should not allow users to adapt the metadata (making it out of sync)

**Comment by mike-podolskiy90** [comment]
@peterdesmet I'm concerned about the taxonomic scope metadata. We can only infer `taxonID` and `scientificName` - what about the rest of the fields? `taxonIDreference` is currently a required field, for example.
Should we keep the forms, but add an "Infer" function so that the user can fill in the missing information?

**Comment by peterdesmet** [comment]
I would still try _not_ to have a form. For `taxonIDreference` we could either:

- Make it non-required
- Have the user provide a single `taxonIDReference` that is used for all

That way the options would still be: `Automatically infer from source data` or `As provided in uploaded datapackage.json`
